tutorials/plot_1_bycycle_philosophy.py

```diff
@@ -60,27 +65,40 @@
+# Because of this last property, these traces have come to be known as "instantaneous amplitude"
+# and "instantaneous phase." And they seem to make a lot of sense, when looking at the raw signal.
 #
 # However, there are some key disadvantages to this analysis that stem from its sine wave basis.
 #
-# 1. Being defined at every point in time gives the illusion that the phase and amplitude estimates are valid at all points in time. However, the amplitude and phase estimates are pretty garbage when there's no oscillation going on (the latter half of the time series above). The "amplitude" and "phase" values are meaningless when no oscillation is actually present. Rather, they are influenced by the other aspects of the signal, such as transients. For this reason, these measures are flaws, and burst detection is very important to help alleviate this issue.
-# 2. This analysis does not capture a potentially important aspect of the data, in that the oscillatory cycles tend to have short rises and longer decays. This is partly because the signal is filtered in a narrow frequency band (using a sine wave basis) that cannot accurately reconstruct nonsinusoidal waveforms. Furthermore, this nonsinusoidal feature will unintuitively bias amplitude and phase estimates (though perhaps negligibly). Furthermore, there are no apparent tools for extracting nonsinusoidal properties using conventional techniques.
+# 1. Being defined at every point in time gives the illusion that the phase and amplitude estimates
+# are valid at all points in time. However, the amplitude and phase estimates are pretty garbage
+# when there's no oscillation going on (the latter half of the time series above). The "amplitude"
+# and "phase" values are meaningless when no oscillation is actually present. Rather, they are
+# influenced by the other aspects of the signal, such as transients. For this reason, these measures
+# are flaws, and burst detection is very important to help alleviate this issue.
+# 2. This analysis does not capture a potentially important aspect of the data, in that the
+# oscillatory cycles tend to have short rises and longer decays. This is partly because the signal
+# is filtered in a narrow frequency band (using a sine wave basis) that cannot accurately
+# reconstruct nonsinusoidal waveforms. Furthermore, this nonsinusoidal feature will unintuitively
+# bias amplitude and phase estimates (though perhaps negligibly). Furthermore, there are no apparent
+# tools for extracting nonsinusoidal properties using conventional techniques.
 #
 #
-# Note that different hyperparameter choices for filters can lead to significant differences in results
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Note that different hyperparameter choices for filters can lead to significant differences in
```

Comment:
Making the line break here breaks it being a title, so I think on this one you can leave it on one line

Reply:
Fixed